### DISCUSSION TITLE
https://chat.openai.com/g/g-4wfedlmN1

### DISCUSSION THREAD

**Discussion by giscus** [lencx/gpthub#20] [Comments]
# https://chat.openai.com/g/g-4wfedlmN1

Discover the Best in Custom GPT at OpenAI's GPT Store – Your Adventure Begins Here!

https://gpthub.nofwl.com/

**Comment by xiyuecangxin** [comment]
卧槽大佬牛逼，制作这种gpt是自己上传代码，还是给它提示词让gpt自己写代码？
